src/Nowruz/modules/general/components/stepperCard/index.tsx

```diff
@@ -91,14 +107,7 @@ export const StepperCard: React.FC<StepperCardProps> = (props) => {
           <Typography variant="h4" component="div" color={variables.color_grey_900}>
             <Box component="h4" display="flex" alignItems="center" sx={{ gap: 0.5 }}>
               {title}
-              {verified ? (
-                <Box component="span" display="flex" alignItems="center" color={variables.color_grey_400}>
-                  <Icon name="shield-tick" color={variables.color_success_500} />
-                  <span className="text-sm font-normal leading-5 text-Gray-light-mode-600">(verified)</span>
-                </Box>
-              ) : (
-                <span className="text-sm font-normal leading-5 text-Gray-light-mode-600">(unverified)</span>
-              )}
+              {verificationStatusContent[verified]}
```

Comment:
Whenever we want to use it for them i will change it.
Because i checked everywhere and `verification` not used 
I prefer to change it according to my requirements and then if the requirement is changed, i will change it
We don't know about educational certificates completely it's better to handle it in future.
Unless you use this component with this property. do you use it?